sql/core/src/main/scala/org/apache/spark/sql/classic/RelationalGroupedDataset.scala

```diff
@@ -479,6 +505,24 @@ class RelationalGroupedDataset protected[sql](
       timeModeStr: String,
       initialState: RelationalGroupedDataset,
       eventTimeColumnName: String): DataFrame = {
+    _transformWithStateInPySpark(
+      func,
+      outputStructType,
+      outputModeStr,
+      timeModeStr,
+      initialState,
+      eventTimeColumnName,
+      TransformWithStateInPySpark.UserFacingDataType.PANDAS)
+  }
+
+  private def _transformWithStateInPySpark(
+      func: Column,
+      outputStructType: StructType,
+      outputModeStr: String,
+      timeModeStr: String,
+      initialState: RelationalGroupedDataset,
+      eventTimeColumnName: String,
+      userFacingDataType: TransformWithStateInPySpark.UserFacingDataType.Value): DataFrame = {
```

Comment:
Should we use a simpler name here maybe ? instead of `userFacingDataType` ?

Reply:
I'm not sure this is complicated - what else you want to propose?

Reply:
Oh nvm - this is not directly exposed to users. Name is different between pandas and non-pandas anyway - so should be fine